bucket/pacmc.json

```diff
@@ -17,6 +18,6 @@
             "java/openjdk"
         ]
     },
-    "homepage": "https://github.com/bluefireoly/pacmc"
+    "homepage": "https://github.com/bluefireoly/pacmc",
+    "notes": "Thank you for installing pacmc, run the 'pacmc' command to get started."
```

Comment:
We assume that people know how to run applications they just installed using Scoop. Scoop will print messages about binaries being added to the `PATH` or in the Start Menu.

```suggestion
```